tasks/install.yml

```diff
@@ -47,12 +47,30 @@
 
 - meta: flush_handlers
```

Comment:
By the way, why is it here? Handlers are usually executed at the end of a play, and flushing them earlier should be used only when necessary.